primitives/blockchain/src/error.rs

```diff
@@ -35,110 +35,150 @@ pub enum ApplyExtrinsicFailed {
 	/// unappliable onto the current block.
 	#[error("Extrinsic is not valid: {0:?}")]
 	Validity(#[from] TransactionValidityError),
+
 	/// This is used for miscellaneous errors that can be represented by string and not handleable.
 	///
 	/// This will become obsolete with complete migration to v4 APIs.
+	#[deprecated(note = "Introduce more typed error variants as needed")]
 	#[error("Extrinsic failed: {0}")]
 	Msg(String),
 }
 
 /// Substrate Client error
 #[derive(Debug, thiserror::Error)]
+#[allow(missing_docs)]
```

Comment:
can we fix this? :)

Reply:
If you look at other impls, I would favour this variant over adding additional `/// copy of error message fmt str` which adds no information and just exists to for process satisfaction purposes :monocle_face:

Reply:
If we go this way, we should remove the existing ones ;)